lib/documents/document.js

```diff
@@ -36,6 +38,8 @@ export class Document extends BaseDocument {
       publishedAt: 1,
       title: 1,
       contentKey: 1,
+      forkedFrom: 1,
+      forkedAtVersion: 1,
```

Comment:
Do you have to publish these fields to the client?

Reply:
I use these fields to toggle the Fork/Merge buttons in the sidebar component. Should I add a boolean field for this? something like `isForked` maybe?